packages/resource-deployment/scripts/upload-files.sh

```diff
@@ -6,7 +6,9 @@
 set -eo pipefail
 
 export jobManagerContainerName="batch-job-manager-script"
+export webAPIScanJobManagerContainerName="batch-web-api-scan-job-manager-script"
```

Comment:
>webAPIScanJobManagerContainerName [](start = 7, length = 33)

webApi

Reply:
API isn't a word, it's initials, so I think it should be API

Reply:
i know. but when you use this in a variable name, we don't do that, so that it is easier to know where the word ends. so, it should be webApiScanJobManagerContainerName

Reply:
let's discuss offline